apps/src/code-studio/components/TeacherContentToggle.js

```diff
@@ -138,7 +138,7 @@ export const mapStateToProps = state => {
     );
   } else if (!state.verifiedInstructor.isVerified) {
     // if not-authorized teacher
-    isLockedLesson = state.progress.lessons.some(
+    isLockedLesson = state.progress.lessons?.some(
```

Comment:
So... we are only in this 'if' block when we have a teacher who is not viewingAs a participant, and is not verified? And the page was stuck because we weren't loading in progress, so there were no progress.lessons to look at? I think I vaguely understand what's going on but the url params are so messy that I'd love to someday rethink the entire teacher panel and how it gets data.

Reply:
Yeah, you have it right. It is very confusing. After some further thought, I'm going to hide the teacher panel in this scenario altogether. It doesn't really make sense to render a teacher panel when viewing a teacher-peers work

Reply:
Heck yeah to hiding the teacher panel- I endorse this message🙌🏼